app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,12 @@
     <meta name="sentry-current-env" content="<%= ENV["SENTRY_CURRENT_ENV"] %>">
   <% end %>
 
+  <script>
```

Comment:
How come this isn't in the conditional anymore?

Reply:
Sorry that should have been made clear in a commit. I'll sort that.

But it's because other pages can add data to the dataLayer and if it's not always there then you can't see whether it's doing what you expect until you are integration.